assets/js/componentRequests.js

```diff
@@ -0,0 +1,21 @@
+jQuery(function () {
+    const columns = [
+      {
+        data: 'tf_label',
+        createdCell: function (td, _cellData, rowData) {
+          const display = `${rowData.tf_label}`.trim()
+          $(td).html(`${rowData.name}</a>`)
+        },
+      },
+      {
+        data: 'tf_label',
+        createdCell: function (td, _cellData, rowData) {
+          const display = `${rowData.tf_label}`.trim()
+          $(td).html(`${rowData.request_processed_status}</a>`)
+        },
+      }
+    ]
+  
+    console.log('columns', columns)
```

Comment:
Try not to check in code with `console.log` because it can get left in.

Reply:
Removed console.log